qa/suites/upgrade/nautilus-x/parallel/1-ceph-install/nautilus.yaml

```diff
@@ -35,6 +36,9 @@ tasks:
     mon.b:
     mon.c:
 - print: "**** done install.upgrade non-client hosts"
+- rgw:
+   - client.0
+- print: "**** done => started rgw client.0"
```

Comment:
qa/tasks/rgw.py stores some global state in ctx.rgw about its endpoints, and other tasks like s3tests/swift/ragweed use those endpoints to connect. if you run two separate rgw tasks, the second one will overwrite the first one's global state. so both clients should be created with the same rgw task:
```yaml
- rgw:
   - client.0
   - client.1
- print: "**** done => started rgw client.0 and client.1"
```